packages/contracts-bedrock/src/L2/GasPriceOracle.sol

```diff
@@ -24,23 +25,50 @@ contract GasPriceOracle is ISemver {
     uint256 public constant DECIMALS = 6;
 
     /// @notice Semantic version.
-    /// @custom:semver 1.2.0
-    string public constant version = "1.2.0";
+    /// @custom:semver 1.3.0
+    string public constant version = "1.3.0";
+
+    // Hardcoded values for the Fjord upgrade calculation
+    int32 private constant COST_INTERCEPT = -42_585_600;
+    uint32 private constant COST_FASTLZ_COEF = 836_500;
+    uint256 private constant MIN_TRANSACTION_SIZE = 100;
```

Comment:
I see that these are inlined into the bytecode meaning that the constant should be inlined into the native source code as well which makes sense to reduce i/o during the hot path